packages/collector/test/tracing/frameworks/fastify/test.js

```diff
@@ -20,7 +20,7 @@ const mochaSuiteFn = supportedVersion(process.versions.node) ? describe : descri
 mochaSuiteFn('tracing/fastify', function () {
   this.timeout(config.getTestTimeout());
 
-  ['1.14.6', '2.15.3', '3.21.5'].forEach(version => {
+  ['1.14.6', '2.15.3', '3.21.5', '4.24.3'].forEach(version => {
```

Comment:
Suggestion: We got rid of this specific version pattern in the past. Could you please adapt it for fastify?
See example https://github.com/instana/nodejs/blob/v2.34.1/packages/collector/test/tracing/database/mongodb/test.js#L23